src/cljs/org/broadinstitute/firecloud_ui/common/table_utils.cljs

```diff
@@ -3,6 +3,7 @@
    [dmohs.react :as react]
    [inflections.core :refer [pluralize]]
    [org.broadinstitute.firecloud-ui.common :as common]
+   [org.broadinstitute.firecloud-ui.common.gcs-file-preview :refer [GCSFilePreviewLink]]
```

Comment:
Just FYI: I would prefer to refer to this as `gcs-file-preview/LinkComponent`, but I haven't been doing that because then when I'm debugging, a component called "LinkComponent" isn't helpful at all. Filed this issue: https://github.com/dmohs/react-cljs/issues/2